### PR TITLE
Update Link

### PR DESCRIPTION
**Pull request by mikeamoss**
Fix broken link from legacy Technet redirect.

### CODE REVIEW

File: WindowsServerDocs/networking/branchcache/BranchCache-Network-Shell-and-Windows-PowerShell-Commands.md [hunk 1]
```diff
@@ -25,7 +25,7 @@ Windows PowerShell and netsh command references are at the following locations.
   
 -   [Netsh Commands for BranchCache in Windows Server 2008 R2](https://technet.microsoft.com/library/dd979561(v=ws.10))  
   
--   [BranchCache Cmdlets in Windows PowerShell](https://technet.microsoft.com/library/hh848392.aspx) for  Windows Server 2012 .  
+-   [BranchCache Cmdlets in Windows PowerShell](https://docs.microsoft.com/en-us/powershell/module/branchcache/?view=winserver2012r2-ps) for  Windows Server 2012 .
```
Comment (ItzLevvie) [inline WindowsServerDocs/networking/branchcache/BranchCache-Network-Shell-and-Windows-PowerShell-Commands.md]:
```suggestion
-   [BranchCache Cmdlets in Windows PowerShell](https://docs.microsoft.com/powershell/module/branchcache/?view=winserver2012r2-ps) for  Windows Server 2012 .  
```

File: WindowsServerDocs/networking/branchcache/BranchCache-Network-Shell-and-Windows-PowerShell-Commands.md [hunk 2]
```diff
@@ -25,7 +25,7 @@ Windows PowerShell and netsh command references are at the following locations.
   
 -   [Netsh Commands for BranchCache in Windows Server 2008 R2](https://technet.microsoft.com/library/dd979561(v=ws.10))
```
Comment (ItzLevvie) [inline WindowsServerDocs/networking/branchcache/BranchCache-Network-Shell-and-Windows-PowerShell-Commands.md]:
```suggestion
-   [Netsh Commands for BranchCache in Windows Server 2008 R2](https://docs.microsoft.com/previous-versions/windows/it-pro/windows-server-2008-R2-and-2008/dd979561(v=ws.10))  
```

Comment (ItzLevvie) [inline WindowsServerDocs/networking/branchcache/BranchCache-Network-Shell-and-Windows-PowerShell-Commands.md]:
```suggestion
-   [Netsh Commands for BranchCache in Windows Server 2008 R2](https://docs.microsoft.com/previous-versions/windows/it-pro/windows-server-2008-R2-and-2008/dd979561(v=ws.10))  
```

**Comment by illfated** [comment]
I concur with the suggested changes. Even if the pages are only available in English (en-us) at this time, it is usually a good idea to keep the links language agnostic, just in case some of them are (or will become) available in other languages (except in the event that any of the links become broken again if the /en-us/ part is removed).

**Comment by eross-msft** [comment]
@ItzLevvie  If you want to commit your changes and fix the conflict, I can merge your changes. I don't have the permissions to commit your changes to your branch.

**Comment by illfated** [comment]
Looks like this PR needs a rewrite to add back the technet link replacement for [Netsh Commands for BranchCache in Windows Server 2008 R2].

**Comment by eross-msft** [comment]
@ItzLevvie @mikeamoss  I believe since you added the change, you must use the **Commit suggestion** button for each of your suggestions. I can't merge anything until your suggestions are part of the actual PR. This must be done in the next 30 days or it will become necessary to close this PR.

**Comment by illfated** [comment]
As mentioned in the other PRs, I am willing to open a new PR to have the changes incorporated in a new PR to get the necessary changes added to the source.

**Comment by eross-msft** [comment]
@illfated  Closing this item. Please open a new PR so we can go ahead and accept these changes.

**Comment by illfated** [comment]
This PR has already been resolved outside of this conversation (see also closed PR **#3428** ).

**Pull request closed without merging**